Modules/FindCUDA.cmake

```diff
@@ -734,15 +767,14 @@ if(NOT CUDA_VERSION VERSION_LESS "5.5")
   cuda_find_library_local_first(CUDA_cudart_static_LIBRARY cudart_static "static CUDA runtime library")
   mark_as_advanced(CUDA_cudart_static_LIBRARY)
 endif()
-if(CUDA_cudart_static_LIBRARY)
-  # Set whether to use the static cuda runtime.
-  option(CUDA_USE_STATIC_CUDA_RUNTIME "Use the static version of the CUDA runtime library if available" ON)
-else()
-  option(CUDA_USE_STATIC_CUDA_RUNTIME "Use the static version of the CUDA runtime library if available" OFF)
-endif()
 
-if(CUDA_USE_STATIC_CUDA_RUNTIME)
-  if(UNIX)
+if(NOT CUDA_USE_STATIC_CUDA_RUNTIME STREQUAL OFF)
+  if(CUDA_cudart_static_LIBRARY)
+    # Set whether to use the static cuda runtime.
+    set(CUDA_USE_STATIC_CUDA_RUNTIME ON CACHE BOOL "Use the static version of the CUDA runtime library if available")
```

Comment:
You should note that the effect of set(VAR value CACHE ...) is only the first invocation ever during configure.  Once it is in the cache, then this call has no effect unless you apply a FORCE argument which will always override the value in the CACHE (despite what the user by want to change via the GUI or some other means).  It is more useful to provide a default value based on the presence of CUDA_cudart_static_LIBRARY like I had before.